easybuild/easyconfigs/c/Clp/Clp-1.17.6-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,37 @@
+# easybuild easyconfig
+# John Dey jfdey@fredhtuch.org fizwit@github
+easyblock = 'ConfigureMake'
+
+name = 'Clp'
+version = '1.17.6'
+
+homepage = 'https://github.com/coin-or/Clp'
+description = """Clp (Coin-or linear programming) is an open-source linear programming solver"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = ['https://github.com/coin-or/Clp/archive/releases']
+sources = ['%(version)s.tar.gz']
+checksums = ['afff465b1620cfcbb7b7c17b5d331d412039650ff471c4160c7eb24ae01284c9']
+
+builddependencies = [
+    ('binutils', '2.35'),
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('CoinUtils', '2.11.4'),
+    ('Osi', '0.108.6'),
+]
```

Comment:
I would suspect that zlib and bzip2 are still required, they tend to sneak in everywhere. Could you check your build log, or `ldd libClp.so` and confirm if they are not present here?